src/domain/projects/templates/showProject.tpl.php

```diff
@@ -197,6 +197,28 @@
                                 </form>
                             </div>
                         </div>
+
+                        <?php // Slack webhook ?>
+                        <h4 class='widgettitle title-light'><span class='iconfa iconfa-leaf'></span>Discord</h4>
+                        <div class='row'>
+                            <div class='col-md-3'>
+                                <img src='<?= BASE_URL ?>/images/discord-logo.png' width='200'/>
```

Comment:
Can you add the discord logo to the images folder?

Reply:
Done! Thank you for catching that.